src/thirdParty/getPluginList.js

```diff
@@ -0,0 +1,5 @@
+import { lib } from '../lib.js';
+
+export default function () {
+  return Object.getOwnPropertyNames(lib);
```

Comment:
it might be nice if you could also get all properties prefixed with `mixins` or any other "type"